frontend/src/concepts/pipelines/content/artifacts/charts/ConfusionMatrix.tsx

```diff
@@ -0,0 +1,185 @@
+              {row.map((cell, cellIndex) => (
+                <td
+                  key={labels[cellIndex] + labels[rowIndex]}
+                  style={{
+                    backgroundColor: getColor(cell),
+                    border: `solid 1px white`,
+                    color: cell / max < 0.6 ? 'black' : 'white',
+                    height: cellSize,
+                    minHeight: cellSize,
+                    minWidth: cellSize,
+                    position: 'relative',
+                    textAlign: 'center',
+                    verticalAlign: 'middle',
+                    width: cellSize,
+                  }}
+                >
+                  {cell}
+                </td>
+              ))}
+            </tr>
+          ))}
+          <tr>
+            <th
+              style={{
+                width: cellSize,
+              }}
+            />
+            {labels.map((label, i) => (
+              <th key={i}>
+                <div
+                  style={{
+                    writingMode: 'vertical-lr',
+                    whiteSpace: 'nowrap',
+                    width: cellSize,
+                  }}
+                >
+                  <Text style={{ transform: `translateX(${size / 4}px) rotate(315deg)` }}>
+                    {label}
+                  </Text>
+                </div>
+              </th>
+            ))}
+          </tr>
+        </tbody>
+      </table>
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'space-between',
+        }}
+      >
+        <div
+          style={{
+            height: 0.75 * data.length * size,
+            background: gradientLegend,
+            borderRight: 'solid 1px #777',
+            marginLeft: 30,
+            minWidth: 10,
+            position: 'relative',
+            width: 10,
+          }}
+        >
+          <div
+            style={{
+              top: 0,
+              borderTop: 'solid 1px #777',
+              left: '100%',
+              paddingLeft: 5,
+              position: 'absolute',
+              width: 5,
+            }}
+          >
+            <span
+              style={{
+                left: 15,
+                position: 'absolute',
+                top: -7,
+              }}
+            >
+              {max}
+            </span>
+          </div>
+          {new Array(5).fill(0).map((_, i) => (
+            <div
+              key={i}
+              style={{
+                top: `${((5 - i) / 5) * 100}%`,
+                borderTop: 'solid 1px #777',
+                left: '100%',
+                paddingLeft: 5,
+                position: 'absolute',
+                width: 5,
+              }}
+            >
+              <span style={{ left: 15, position: 'absolute', top: -7 }}>
+                {Math.floor((i / 5) * max)}
+              </span>
+            </div>
+          ))}
+        </div>
+        <div
+          style={{
```

Comment:
A lot of the inline styles in this file can be achieved by using PF util classnames.

Reply:
The styles that aren't available via PF classnames, seems like there's enough to be pushed inside of separate stylesheets for the ConfusionMatrix in the form of our own classnames.